### ISSUE TITLE
Add integration test for recreating divide by zero error in ParquetCachedBatchSerializer

### ISSUE THREAD

**Issue by razajafri** [NVIDIA/spark-rapids#8397]
Serializer can run into a scenario where we try to process an empty batch where estimating row size results in a divide by zero error

**Comment by razajafri** [comment]
@jlowe can we close this issue now that we have https://github.com/NVIDIA/spark-rapids/blob/618492d6773b36ce96f41361bda0bbd18d5cd0a6/integration_tests/src/main/python/cache_test.py#L357

**Comment by jlowe** [comment]
Yes, assuming that is what reproduced the original divide by zero error.
